package.json

```diff
@@ -2,20 +2,18 @@
     "name": "streamr-monorepo",
     "private": true,
     "scripts": {
```

Comment:
- The`eslint` command could run just eslint, not `manypkg check`? (include to be part of the `check` command maybe?)
- The `fix` command should not call `manypkg check` if that doesn't do any fixing? We could maybe rename the command to `eslint-fix`
- As we do re-ordering of scripts, we could e.g.:
  - put all mostly used script first: `build`, `eslint`, `check` 
  - then all installing command (`bootstrap` etc.)
  - then all cleanup command
  - then other command

Reply:
Discussed and appropriate changes applied.